apps/tooljet/app.env

```diff
@@ -0,0 +1,42 @@
+TOOLJET_HOST=
+
+LOCKBOX_MASTER_KEY=70726290922441785304399601987729268981756953900892
```

Comment:
These keys should be generated randomly ideally. There is a guide [here](https://www.runtipi.io/docs/contributing/adding-a-new-app#using-a-template-in-your-data-folder) in order to use template files. I can help you with an example if needed